orchagent/portsorch.cpp

```diff
@@ -7742,19 +7780,25 @@ void PortsOrch::createPortBufferPgCounters(const Port& port, string pgs)
 {
     SWSS_LOG_ENTER();
 
-    /* Create the PG map in the Counter DB */
-    /* Add stat counters to flex_counter */
-    vector<FieldValueTuple> pgVector;
-    vector<FieldValueTuple> pgPortVector;
-    vector<FieldValueTuple> pgIndexVector;
-
     auto toks = tokenize(pgs, '-');
     auto startIndex = to_uint<uint32_t>(toks[0]);
     auto endIndex = startIndex;
     if (toks.size() > 1)
     {
         endIndex = to_uint<uint32_t>(toks[1]);
     }
+    addPortBufferPgCounters(port, startIndex, endIndex);
```

Comment:
@saksarav-nokia where is createPortBufferPgCounters function being called ?

Reply:
> @saksarav-nokia where is createPortBufferPgCounters function being called ?

createPortBufferPgCounters was existing function and it is called from BufferOrch::processPriorityGroupPost in bufferorch.cpp